app/dashboard/static/js/app/view-builds-id.2020.5.js

```diff
@@ -625,14 +623,23 @@ require([
             str += job;
             str += '/';
             aNode.setAttribute('href', str);
-            aNode.appendChild(html.boot());
+            aNode.appendChild(html.tree());
 
             html.replaceContent(document.getElementById('tree'), docFrag);
 
             // Branch.
+            branchNode = html.tooltip();
+            branchNode.title =
+                "Branch reports for &#171;" + job + "&#187; - " + branch;
```

Comment:
This should be "Test reports" rather than "Branch reports", for test reports from a branch:

"Test reports for branch «master»"